.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "voorjaar.windicss-intellisense",
+    "lokalise.i18n-ally",
+    "johnsoncodehk.volar",
```

Comment:
404 for johnsoncodehk:
`https://open-vsx.org/extension/johnsoncodehk/vscode-typescript-vue-plugin`

Reply:
... https://github.com/open-vsx/publish-extensions/pull/436

Reply:
well yes it should be added - but I can not find it https://open-vsx.org/?search=Volar

Reply:
They have a bug in their build job. Will try to create a PR with a fix.

Reply:
https://github.com/open-vsx/publish-extensions/pull/525